src/commands/pardon.js

```diff
@@ -0,0 +1,259 @@
+const Discord = require('discord.js');
+const { SlashCommandBuilder } = require('@discordjs/builders');
+const Warnings = require('../models/warnings');
+const Kicks = require('../models/kicks');
+const Bans = require('../models/bans');
+const util = require('../util');
+
+/**
+ *
+ * @param {Discord.CommandInteraction} interaction
+ */
+async function pardonHandler(interaction) {
+    await interaction.deferReply({
+		ephemeral: true
+	});
+
+	const guild = await interaction.guild.fetch();
+	const executingMember = await interaction.member.fetch();
+	const executor = executingMember.user;
+    const type = interaction.options.getString('type');
+	const users = interaction.options.getString('users');
+    const reason = interaction.options.getString('reason');
+
+    const userIds = [...new Set(Array.from(users.matchAll(Discord.MessageMentions.USERS_PATTERN), match => match[1]))];
+
+    const image = new Discord.MessageAttachment('./src/images/mod/mod-pardon.png');
+    // Initialize all of the embeds
+    const pardonedListEmbed = new Discord.MessageEmbed();
+    const eventLogEmbed = new Discord.MessageEmbed();
+    const pardonDmEmbed = new Discord.MessageEmbed();
+
+	for (const userId of userIds) { 
+        const member = await interaction.guild.members.fetch(userId);
+		const user = member.user;
+
+        if (type === 'warn_type') { // If chosen warn within options
+            let { userWarns } = await Warnings.findAndCountAll({ // Grab all warns
+                where: {
+                    user_id: member.id
+                }
+            });
+            if (userWarns < 1 || isNaN(userWarns)) { // User has no warns
+                throw new Error('This user has no warns, did you put in the wrong info?');
```

Comment:
It would be nicer if this was a bit more professional sounding. Something like
```suggestion
                throw new Error('This user has no warnings. Ensure you are using the correct user');
```

Reply:
Added suggestion.